src/components/SaveIndicator.tsx

```diff
@@ -0,0 +1,31 @@
+import { useIntl } from "react-intl";
+import { SaveStatus } from "../abstractions/common/save-status";
+
+export function SaveIndicator({ saveStatus }: { saveStatus: SaveStatus }) {
+  const intl = useIntl();
+
+  const { statusClassName, statusText, statusDescription } =
+    saveStatus === "idle"
+      ? {
+          statusClassName: "save-indicator-idle",
```

Comment:
@GusBaamonde what do you think should be the name of the component idle status class in place of `save-indicator-idle`?